package/network/utils/iproute2/Makefile

```diff
@@ -8,12 +8,16 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=iproute2
-PKG_VERSION:=5.10.0
+PKG_VERSION:=5.11rc1
 PKG_RELEASE:=1
 
-PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.xz
-PKG_SOURCE_URL:=@KERNEL/linux/utils/net/iproute2
-PKG_HASH:=a54a34ae309c0406b2d1fb3a46158613ffb83d33fefd5d4a27f0010237ac53e9
+PKG_SOURCE_VERSION:=5a37254b71249bfb73d44d6278d767a6b127a2f9
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://git.kernel.org/pub/scm/network/iproute2/iproute2.git
+PKG_MIRROR_HASH:=
+#PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.xz
+#PKG_SOURCE_URL:=@KERNEL/linux/utils/net/iproute2
+#PKG_HASH:=a54a34ae309c0406b2d1fb3a46158613ffb83d33fefd5d4a27f0010237ac53e9
```

Comment:
Drop the comment?

Reply:
Sure, I intend to, but left this is as placeholder/reminder about the final version (release or pre-release) being undetermined, much like the empty `PKG_MIRROR_HASH:=`. For me, the PR has big improvements, but I have no insight into when/if a 5.11 iproute2 release will drop early enough before the 21.02 release to get this in. That's one aspect of feedback I'm looking for from the maintainer/past committers.

What's your opinion? Aim to merge this pre-release and update later? Or wait and see (but for how long)? Would it be better to mark this "draft" while we decide?

Reply:
Okay, I only had a quick look and didn't really catch that in the first place.
Indeed, marking as draft will be an indicator for something that is not to be merged yet (although it won't have any practical effect for the core repo where stuff is picked from PRs).
I will put the "blocked" tag, as that's how I understand the current status then?

Reply:
That should be fine as we only want to prevent a "drive-by" merge before hearing from maintainers. The current situation is a bit strange: usually the iproute2 release trails the kernel release by only 3-4 days.